taxcalc/utils.py

```diff
@@ -197,14 +196,19 @@ def groupby_income_bins(df, bins=None, right=True):
             indicate (1,2], (2,3], (3,4].
 
     """
-
-    if not bins:
+    if compare_with == "tpc":
```

Comment:
On second thought, the pre-existing bins are slightly preferable: 
"preexisting bins": [-1e14, 0, 9999, 19999, 29999, 39999, 49999, 74999, 99999, 200000, 1e14]